### PR TITLE
Code of conduct

### LINKED ISSUE #2: Code of Conduct
We should have a code of conduct. http://confcodeofconduct.com/ is a great starting point

### PR DESCRIPTION
**Pull request by chadfawcett**
@stefaniebutland and @kluft Can I get you to review this please?

Resolves #2

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
+# Meetup Code of Conduct
+
+All organizers, attendees, speakers, sponsors and volunteers at Kamloops R
+Meetup events are required to agree to uphold the following code of conduct.
+Kamloops R Meetup event organizers and/or Kamloops Innovation staff will enforce
```
Comment (chadfawcett) [inline CODE_OF_CONDUCT.md:5]: Have we confirmed with Beth or Kimberely about having KIC help in enforcing the Code of Conduct?

File: CODE_OF_CONDUCT.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+otherwise create a sexualized environment.
+
+If a participant engages in harassing behavior, the event organizers or Kamloops
+Innovation staff may take any action they deem appropriate, including warning
+the offender or expulsion from the event with no refund.
+
+If you are being harassed, notice that someone else is being harassed, or have
+any other concerns, please contact a member of the organizing team immediately.
+If you have concerns about a member of the organizing team, please email
+beth@kamloopsinnovation.ca.
```
Comment (chadfawcett) [inline CODE_OF_CONDUCT.md:45]: Same here... I don't want to use Beth as a contact if we haven't confirmed it with her first.

**Comment by stefaniebutland** [comment]
I haven’t spoken with either about it

File: CODE_OF_CONDUCT.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+immediately.
+
+Sponsors are also subject to the anti-harassment policy. In particular, sponsors
+should not use sexualized images, activities, or other material. Sponsor staff
+(including volunteers) should not use sexualised clothing/uniforms/costumes, or
+otherwise create a sexualized environment.
+
+If a participant engages in harassing behavior, the event organizers or Kamloops
+Innovation staff may take any action they deem appropriate, including warning
+the offender or expulsion from the event with no refund.
```
Comment (kaseyluft) [inline CODE_OF_CONDUCT.md:40]: I suppose we could remove the "with no refund" from this line.

**Review by kaseyluft — COMMENTED** [review]
added one comment. But I do agree we should talk with Beth or Kimberley before adding their names/email.

File: CODE_OF_CONDUCT.md (hunk 3 above)
Comment (chadfawcett) [reply CODE_OF_CONDUCT.md:40]: Yeah it's meant to be generic. This way it covers the case of if we charge for an event in the future.

**Comment by chadfawcett** [comment]
I've asked Beth if she would be willing to be a person of Contact on behalf of KIC for the Code of Conduct.

**Comment by chadfawcett** [comment]
Beth said the use of Kamloops Innovation in the CoC is okay, and she said she is okay with being the contact.

**Pull request merged**